OpenTaiko/src/Stages/04.Config/CActConfigList.cs

```diff
@@ -606,14 +562,6 @@ public void tEnter押下() {
 			#endregion
 			  else {
 				// #27029 2012.1.5 from
```

Comment:
This comment seems to be for the removed code. It should also be removed.
https://osdn.net/projects/dtxmania/ticket/27029